plugins/yaml/resources/META-INF/plugin.xml

```diff
@@ -113,4 +112,4 @@
     </extensionPoint>
   </extensionPoints>
 
-</idea-plugin>
```

Comment:
Why do you change the last line? :)

Reply:
Sorry my bad. Editor configured to add missing eof newlines. Will revert this change.